1-js/03-code-quality/06-polyfills/article.md

```diff
@@ -1,25 +1,21 @@
 
 # Polyfills
 
-The JavaScript language steadily evolves. New proposals to the language appear regularly, they are analyzed and, if considered worthy, are appended to the list at <https://tc39.github.io/ecma262/> and then progress to the [specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
+The JavaScript language steadily evolves. New features requests are regularly [proposed](https://github.com/tc39/proposals) to the language and are vetted through a [4 part staging process](https://tc39.github.io/process-document/). If considered worthy, these are added to the evergreen spec at <https://tc39.github.io/ecma262/> and then get standardized to the [yearly release specification](http://www.ecma-international.org/publications/standards/Ecma-262.htm).
 
-Teams behind JavaScript engines have their own ideas about what to implement first. They may decide to implement proposals that are in draft and postpone things that are already in the spec, because they are less interesting or just harder to do.
-
-So it's quite common for an engine to implement only the part of the standard.
-
-A good page to see the current state of support for language features is <https://kangax.github.io/compat-table/es6/> (it's big, we have a lot to study yet).
+A way to see the current state of support for language features is <https://kangax.github.io/compat-table/es6/> (it's big, we have a lot to study yet).
 
 ## Babel
 
-When we use modern features of the language, some engines may fail to support such code. Just as said, not all features are implemented everywhere.
+When we use modern features of the language, some engines may fail to support such code. Engines will not usually implement any proposal before Stage 3 since the feature is in flux, and shipping a feature in a browser takes a lot of work.
```

Comment:
This change requires the reader to get familiar with the proposals evaluation process. Like "what is stage 3?" We shouldn't require it.

The original phrase is correct.